File: Service_eHospital_DongNai_A/Implement/DongNai_AService.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class DongNai_AService : BaseService<DongNai_A, IDongNai_ARepository>
    , IDongNai_AService
    {
    private readonly IDongNai_ARepository _DongNai_ARepository;
    public DongNai_AService(IDongNai_ARepository DongNai_ARepository) : base(DongNai_ARepository)
    {
    _DongNai_ARepository = DongNai_ARepository;
    }
    }
    }

File: Report/src/app/shared/eHospital_DongNai_A/DongNai_A.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { DongNai_A } from './DongNai_A.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class DongNai_AService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'MaBenhAn'];

    List: DongNai_A[] | undefined;
    ListFilter: DongNai_A[] | undefined;
    FormData!: DongNai_A;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "DongNai_A";
    }
}

